frontend/src/components/archive/CreateArchiveDialog.tsx

```
@@ -0,0 +1,202 @@
+              type="text"
+              value={archiveName}
+              onChange={(e) => setArchiveName(e.target.value)}
+              placeholder="e.g., October 2025"
+              className="w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500"
+              disabled={isLoading}
+              maxLength={100}
+              required
+              aria-invalid={!!error}
+              aria-describedby={error ? 'archive-error archive-warning' : 'archive-warning'}
+              aria-label="Enter a name for this payment archive"
+              autoFocus
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)